shared/actions/chat2/index.tsx

```diff
@@ -3408,97 +3384,39 @@ function* chat2Saga(): Saga.SagaGenerator<any, any> {
-  )
+  yield* Saga.chainAction2(Chat2Gen.selectConversation, loadCanUserPerform, 'loadCanUserPerform')
+  yield* Saga.chainAction2(Chat2Gen.selectConversation, loadTeamForConv, 'loadTeamForConv')
 
   // Giphy
-  yield* Saga.chainAction<Chat2Gen.UnsentTextChangedPayload>(
-    Chat2Gen.unsentTextChanged,
-    unsentTextChanged,
-    'unsentTextChanged'
-  )
-  yield* Saga.chainAction<Chat2Gen.GiphySendPayload>(Chat2Gen.giphySend, giphySend, 'giphySend')
+  yield* Saga.chainAction2(Chat2Gen.unsentTextChanged, unsentTextChanged, 'unsentTextChanged')
+  yield* Saga.chainAction2(Chat2Gen.giphySend, giphySend, 'giphySend')
 
-  yield* Saga.chainAction<Chat2Gen.UnfurlResolvePromptPayload>(
-    Chat2Gen.unfurlResolvePrompt,
-    unfurlResolvePrompt,
-    'unfurlResolvePrompt'
-  )
-  yield* Saga.chainAction<Chat2Gen.UnfurlResolvePromptPayload>(
-    Chat2Gen.unfurlResolvePrompt,
-    unfurlDismissPrompt,
-    'unfurlDismissPrompt'
-  )
-  yield* Saga.chainAction<Chat2Gen.UnfurlRemovePayload>(Chat2Gen.unfurlRemove, unfurlRemove, 'unfurlRemove')
+  yield* Saga.chainAction2(Chat2Gen.unfurlResolvePrompt, unfurlResolvePrompt, 'unfurlResolvePrompt')
+  yield* Saga.chainAction2(Chat2Gen.unfurlResolvePrompt, unfurlDismissPrompt, 'unfurlDismissPrompt')
+  yield* Saga.chainAction2(Chat2Gen.unfurlRemove, unfurlRemove, 'unfurlRemove')
 
-  yield* Saga.chainAction<Chat2Gen.PreviewConversationPayload>(
-    Chat2Gen.previewConversation,
-    previewConversationTeam
-  )
-  yield* Saga.chainAction<Chat2Gen.PreviewConversationPayload>(
-    Chat2Gen.previewConversation,
-    previewConversationPersonMakesAConversation
-  )
-  yield* Saga.chainAction<Chat2Gen.OpenFolderPayload>(Chat2Gen.openFolder, openFolder)
+  yield* Saga.chainAction2(Chat2Gen.previewConversation, previewConversationTeam)
+  yield* Saga.chainAction2(Chat2Gen.previewConversation, previewConversationPersonMakesAConversation)
+  yield* Saga.chainAction2(Chat2Gen.openFolder, openFolder)
 
   // On login lets load the untrusted inbox. This helps make some flows easier
-  yield* Saga.chainAction<ConfigGen.BootstrapStatusLoadedPayload>(
-    ConfigGen.bootstrapStatusLoaded,
-    startupInboxLoad,
-    'startupInboxLoad'
-  )
+  yield* Saga.chainAction2(ConfigGen.bootstrapStatusLoaded, startupInboxLoad, 'startupInboxLoad')
 
-  yield* Saga.chainAction<ConfigGen.BootstrapStatusLoadedPayload>(
-    ConfigGen.bootstrapStatusLoaded,
-    startupUserReacjisLoad,
-    'startupUserReacjisLoad'
-  )
+  yield* Saga.chainAction2(ConfigGen.bootstrapStatusLoaded, startupUserReacjisLoad, 'startupUserReacjisLoad')
 
-  yield* Saga.chainAction<Chat2Gen.UpdateUserReacjisPayload>(
-    Chat2Gen.updateUserReacjis,
-    onUpdateUserReacjis,
-    'onUpdateUserReacjis'
-  )
+  yield* Saga.chainAction2(Chat2Gen.updateUserReacjis, onUpdateUserReacjis, 'onUpdateUserReacjis')
 
   // Search handling
-  yield* Saga.chainAction<SearchGen.UserInputItemsUpdatedPayload>(
-    SearchConstants.isUserInputItemsUpdated('chatSearch'),
```

Comment:
this form is disallowed now, and its the singleton in the app. turns into a simple 1 liner in the handler anyways